contracts/FlowCallbackScheduler.cdc

```diff
@@ -528,9 +532,12 @@ access(all) contract FlowCallbackScheduler {
             return nextID
         }
 
-        /// get status of the scheduled callback, if the callback is not found Unknown is returned.
+        /// get status of the scheduled callback
+        /// @param id: The ID of the callback to get the status of
+        /// @return Status: The status of the callback, if the callback is not found Unknown is returned.
         access(all) view fun getStatus(id: UInt64): Status? {
```

Comment:
We have a bug here... the problem is that if we query for ID that is bigger than actually executed IDs it will return executed status. We must add another field that keeps track of latest executed ID, then we need to add a check `if id > latestExecutedID return Unknown`